amundsen_application/static/js/ducks/user/reducer.ts

```diff
@@ -2,27 +2,48 @@ import {
   GetCurrentUser,
   GetCurrentUserRequest,
   GetCurrentUserResponse,
-  CurrentUser
+  GetUser,
+  GetUserRequest,
+  GetUserResponse,
+  CurrentUser, User
 } from './types';
 
-type UserReducerAction = GetCurrentUserRequest | GetCurrentUserResponse;
+type UserReducerAction =
+  GetCurrentUserRequest | GetCurrentUserResponse |
+  GetUserRequest | GetUserResponse ;
 
 export interface UserReducerState {
   currentUser: CurrentUser;
+  profilePageUser: User;
```

Comment:
I think `CurrentUser` might be mistaken for the current user being rendered, which is why you might have made `profilePageUser` so explicit. What do you think about disambiguating the users with something like

`currentUser -> loggedInUser`
 `profilePageUser -> profileUser or userProfile`

This can also be a TODO

Reply:
I like those names. Will go ahead and make the change.